.travis.yml

```diff
@@ -6,10 +6,7 @@ matrix:
   include:
     - go: 1.10.x
       install:
-        - go get github.com/stretchr/testify/assert
```

Comment:
Instead of introducing a new Makefile, which doesn't do much, how about adding a `before_install block` that is common to all blocks? https://docs.travis-ci.com/user/customizing-the-build/

Reply:
Won't that block be executed for the go module builds as well? I didn't add one because I don't wan't a `go get` to interfere or pollute the namespace before `go mod download` is executed.

Reply:
Ah, you're right. Good thinking.